weed/server/filer_grpc_server_sub_meta.go

```diff
@@ -39,12 +39,10 @@ func (fs *FilerServer) SubscribeMetadata(req *filer_pb.SubscribeMetadataRequest,
 	var readInMemoryLogErr error
 
 	for {
-
 		glog.V(4).Infof("read on disk %v aggregated subscribe %s from %+v", clientName, req.PathPrefix, lastReadTime)
-
 		processedTsNs, readPersistedLogErr = fs.filer.ReadPersistedLogBuffer(lastReadTime, eachLogEntryFn)
 		if readPersistedLogErr != nil {
-			return fmt.Errorf("reading from persisted logs: %v", readPersistedLogErr)
+			glog.V(4).Infof("read on disk %v subscribe %s from %+v: %v", clientName, req.PathPrefix, lastReadTime, readPersistedLogErr)
```

Comment:
the error could be a retry-able error. Just skipping it may cause more problems later.